scripts/create-kubestellar-demo-env.sh

```diff
@@ -17,16 +17,27 @@
 
 set -e
 
```

Comment:
I strongly recommend including `set -x`, either unconditionally or controlled by a command-line option. This script is likely to be used by newbies, and we will get bug reports, and those will be much more comprehensible with the commands echoed.

Reply:
too noisy - can be done by hand if needed

Reply:
That is labor intensive. That is why our other scripts have a `-X` option on the command line.

Not a blocker.